lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2611,3 +2613,25 @@ definitions:
         pattern: '^(\d{6})$'
         example: '176148'
         x-nullable: false
+
+  paginated_enodebs:
+    description: Page of enodebs
+    properties:
+      next_page_token:
+        $ref: './orc8r-swagger-common.yml#/definitions/next_page_token'
+      enodebs:
+        additionalProperties:
+          $ref: '#/definitions/enodeb'
+          x-nullable: true
```

Comment:
1. Nullable=true is the default
2. Idk why we would need to make this nullable. I believe we can just remove this line to no effect (and also remove it from subs_v2)